docs/reference/commandline/node_ls.md

```diff
@@ -145,17 +145,19 @@ Placeholder      | Description
 `.Status`        | Node status
 `.Availability`  | Node availability ("active", "pause", or "drain")
 `.ManagerStatus` | Manager status of the node
+`.TLSStatus`     | TLS status of the node (whether it is signed by/trusts the current swarm root CA or if it needs a cert rotation)
```

Comment:
Thanks for having a look @mstanleyjones!  Have tried to shorten this to be no longer than the description for `.Self`.